src/Amazon.AspNetCore.Identity.Cognito/CognitoKeyNormalizer.cs

```diff
@@ -37,7 +37,7 @@ public string Normalize(string key)
         }
 #endif
 
-#if NETCOREAPP_3_0
+#if NETCOREAPP_3_1
```

Comment:
As mentioned in the csproj we should be using the built in preprocessor symbols here and in the other files: NETCOREAPP3_1

Reply:
Fixed